lib/tpm2_policy.c

```diff
@@ -200,6 +200,81 @@ bool tpm2_policy_build_pcr(TSS2_SYS_CONTEXT *sapi_context,
     return result;
 }
 
+bool tpm2_policy_build_policyauthorize(TSS2_SYS_CONTEXT *sapi_context,
+    tpm2_session *policy_session, const char *to_authorize_policy_file,
```

Comment:
just go one per line, since that is what you do with the other 3 paths. Shorten these names up, too long.